py/picca/io.py

```diff
@@ -340,7 +340,7 @@ def read_from_spec(in_dir,thid,ra,dec,zqso,plate,mjd,fid,order,mode,log=None,pk1
         for meta in thids[t]:
             r,d,z,p,m,f = meta.ra,meta.dec,meta.zqso,meta.plate,meta.mjd,meta.fid
             try:
-                fin = in_dir + "/{}/{}-{}-{}-{:04d}.fits".format(p,mode,p,m,fid)
```

Comment:
nope, `str(fid)` shouldn't work as then it can't do the  int->str conversion from the format specifier

Reply:
this is also the reason why travis failed below

Reply:
Ok, I just fixed that. Thanks.